OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -2058,6 +2058,13 @@ internal static void UpgradeActorRules(int engineVersion, ref List<MiniYamlNode>
 						node.Key = "RotationPaletteEffect";
 				}
 
+				if (engineVersion < 20150907 && depth == 2 && node.Key == "Range")
+				{
+					if (parentKey == "DetectCloaked" || parentKey == "JamsMissiles" || parentKey == "JamsRadar"
```

Comment:
Considering how many times this may be called (for every level 2 `Range` node), creating a `string[]` may slow it down; however, putting each comparison on its own line should be just as readable.